frontend-too/src/App.js

```diff
@@ -1,42 +1,26 @@
-import React, { useState } from 'react';
-import { Form, Button } from 'react-bootstrap';
-import AddMessage from './components/AddMessage';
-import ChatMessages from './components/ChatMessages';
+import React from 'react';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
```

Comment:
you'll need to add `react-router-dom` to package.json ; you can run `npm install react-router-dom`